packages/sns/src/converters/governance.converters.spec.ts

```diff
@@ -0,0 +1,264 @@
+import { Principal } from "@dfinity/principal";
+import type { Action as ActionCandid } from "../../candid/sns_governance";
+import type { Action } from "../types/actions";
+import { toActionOptional } from "./governance.converters";
+
+describe("governance converters", () => {
+  describe("toActionOptional", () => {
+    const mockPrincipal = Principal.fromText("aaaaa-aa");
+
+    it("converts ManageNervousSystemParameters action", () => {
+      const action: ActionCandid = {
+        ManageNervousSystemParameters: {
+          default_followees: [
+            {
+              followees: [
+                [BigInt(2), { followees: [{ id: new Uint8Array(0) }] }],
+              ],
+            },
+          ],
+          max_dissolve_delay_seconds: [BigInt(2)],
+          max_dissolve_delay_bonus_percentage: [BigInt(2)],
```

Comment:
Please use different values for different fields so you actually catch it if you mix them up in the conversion.
Use constants or something to make it easy to see the right value is used in the expectation.

Reply:
done